### DISCUSSION TITLE
Error in C Debug

### DISCUSSION THREAD

**Discussion by programonauta** [VSCodium/vscodium#979] [Q&A]
Hi,

I've just installed VS Codium with [C/C++ Runner](https://open-vsx.org/extension/franneck94/c-cpp-runner) Extension, but when I try to debug any C program, I get this error:

![image](https://user-images.githubusercontent.com/6033539/151169710-827ef7f2-761e-4683-986d-ab817c714491.png)

I'm using Debian 11, gdb and gcc with versions:

```
$ gdb --version
GNU gdb (Debian 10.1-1.7) 10.1.90.20210103-git
Copyright (C) 2021 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

$ gcc --version
gcc (Debian 10.2.1-6) 10.2.1 20210110
Copyright (C) 2020 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

The extension author [said](https://github.com/franneck94/Vscode-C-Cpp-Runner/issues/53#issuecomment-1019328945) that "debugging is then not possible in vscodium, since this extension needs the Microsoft's C++ extension for the full feature plate".

So is there any other extension that I could use to debug my C programs?

**Comment by daiyam** [comment]
You could download and install the missing `C/C++` extension from https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools

**Reply by programonauta** [reply]
Hi @daiyam, The C/C++ from MS does not run on VS Codium due contains telemetry/tracking.

I'm looking for any other extension able to debug my C programs.
